web/client/examples/layertree/app.jsx

```diff
@@ -55,16 +56,18 @@ const getGroup = (group, allLayers) => {
         }, []);
 
     return assign({}, {
+        id: groupName,
```

Comment:
In layers reducer deepChange function is used both for layers and group, now it checks for a generic node id not name